index.html

```diff
@@ -77,7 +77,165 @@ <h3 class="username">Karma</h3>
         </div>
     </div>
     <div class="content">
-      <h3>content</h3>
+      <div class="content-header">
+        <h2 class="content-header-item">Home</h2>
+        <i class="fa fa-certificate" title="Top Tweets"></i>
+      </div>
+      <div class="what-happening-box">
+        <div class="tweet-box">
+          <a href="#" class="user-card">
+            <img src="images/avatar.png" alt="avatar" class="user-avatar"/>
+          </a>
+          <div class="text-privacy-container">
+            <textarea name="tweet" class="tweet-field" cols="40" rows="1" placeholder="What's happening?" required></textarea>
+            <div class="hint">
+              <a href="#"  class="hint-message">
+                  <i class="fa fa-globe"></i> Everyone can reply
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="options">
+          <div class="icons">
+            <a href="#" class="icon" title="Media"><i class="fa fa-image" ></i></a>
+            <a href="#" class="icon" title="GIF"><i class="fa fa-upload" ></i></a>
+            <a href="#" class="icon" title="Poll"><i class="fa fa-bars" ></i></a>
+            <a href="#" class="icon" title="Emoji"><i class="fa fa-comment" ></i></a>
+            <a href="#" class="icon" title="Schedule"><i class="fa fa-file" ></i></a>  
+          </div>     
+          <button class="status" disabled>Tweet</button>
+        </div>
+      </div>
+      <div class="tweet-template">
+        <a href="#" class="user-card">
+          <img src="images/avatar.png" alt="avatar" class="user-avatar"/>
+        </a>
+        <div class="tweet-template-container">
+          <div class="tweet-options">
+            <div class="tweet-info">
+              <div class="user-container">
+                <h3 class="username">Medhat Dawoud </h3>
```

Comment:
remove the spacing around text in the content, spaces should be using css not user input, so instead of `Medhat Dawoud ` remove this last space, and same for the below one, then check the design